src/Operator/Base.jl

```diff
@@ -434,6 +437,7 @@ function getelementmatrix(operator::Operator)
                 for j = 1:numberfieldsin[i]
                     # run user weak form function
                     weakforminputs[i][j] = 1.0
+                    #@show weakforminputs
                     outputs = operator.weakform(weakforminputs...)
                     weakforminputs[i][j] = 0.0
 
```

Comment:
@jeremylt These outputs get unpacked in the lines below. These just need to be consistent with what qfunctions are expected to return. Did you have a vision for that?

Reply:
Actually, the error I am having when running the demo notebook goes to the line
```julia
# run user weak form function
weakforminputs[i][j] = 1.0
outputs = operator.weakform(weakforminputs...)
weakforminputs[i][j] = 0.0
```